README.md

```diff
@@ -453,6 +453,8 @@ Usage of build/bin/sm-server:
     	Max number of attempts for a recoverable error to be retried against the Redis cluster (default 3)
   -notifications string
     	The name of the notification topic in SQS to publish events' outcomes to; if not specified, no outcomes will be published
+  -acks string
```

Comment:
this is a nit, but currently the message here and the one in the app do not match